composer.json

```diff
@@ -9,10 +9,11 @@
     "license": "MIT",
     "require": {
         "php": "^8.2",
-        "sylius/sylius": "2.0.*"
+        "sylius/sylius": "^2.0",
+        "symfony/runtime": "^7.3"
```

Comment:
Runtime again?
Just add it to your dev dependency, why do you keep putting it in require? It's not needed at all.